src/lib/modules/Document/DocumentListEntry/DocumentListEntry.js

```diff
@@ -109,6 +110,10 @@ export default class DocumentListEntry extends Component {
             <LiteratureTitle title={this.metadata.title} />
           </Item.Header>
           <Item.Meta>
+            <Overridable
```

Comment:
this is named after authors, but it is before, no? Is it the right place?

Reply:
I renamed it